Source/FieldSolver/FiniteDifferenceSolver/ComputeDivE.cpp

```diff
@@ -195,4 +203,55 @@ void FiniteDifferenceSolver::ComputeDivECylindrical (
 
 }
 
-#endif // corresponds to ifndef WARPX_DIM_RZ
+#elif defined(WARPX_DIM_RSPHERE)
+
+template<typename T_Algo>
+void FiniteDifferenceSolver::ComputeDivESpherical (
+    ablastr::fields::VectorField const & Efield,
+    amrex::MultiFab& divEfield
+)
+{
+    // Loop through the grids, and over the tiles within each grid
+#ifdef AMREX_USE_OMP
+#pragma omp parallel if (amrex::Gpu::notInLaunchRegion())
+#endif
+    for ( MFIter mfi(divEfield, TilingIfNotGPU()); mfi.isValid(); ++mfi ) {
+
+        // Extract field data for this grid/tile
+        const Array4<Real> divE = divEfield.array(mfi);
+        Array4<Real> const& Er = Efield[0]->array(mfi);
+
+        // Extract stencil coefficients
+        Real const * const AMREX_RESTRICT coefs_r = m_stencil_coefs_r.dataPtr();
+        auto const n_coefs_r = static_cast<int>(m_stencil_coefs_r.size());
+
+        // Extract cylindrical specific parameters
+        Real const dr = m_dr;
+        Real const rmin = m_rmin;
+
+        // Extract tileboxes for which to loop
+        Box const& tdive  = mfi.tilebox(divEfield.ixType().toIntVect());
+
+        // Loop over the cells and update the fields
+        amrex::ParallelFor(tdive,
+
+            [=] AMREX_GPU_DEVICE (int i, int j, int /*k*/){
+                Real const r = rmin + i*dr; // r on a nodal grid (F is nodal in r)
```

Comment:
Should `F` be `divE` in this comment?

Reply:
No, this was left over from code copied. Now removed.